src/main.rs

```diff
@@ -214,7 +217,10 @@ fn main() {
                 .add_handler(LookupProfiles::new())
                 .add_handler(LookupFriends::new())
                 .add_handler(DumbAutoKick)
-                .add_handler(WebAPIHandler::new());
+                .add_handler(WebAPIHandler::new())
+                .add_handler(BroadcastableEvent::new());
+                // .add_handler(SseBroadcastMessages)
+                // .add_handler(SseBroadcastVotes::new());
```

Comment:
Don't forget to remove the commented out stuff if it's gone for good.

Reply:
:clown_face: